tests/Unit/Core/Module/ModuleManagerTest.php

```diff
@@ -77,10 +77,11 @@ public function setUp(): void
                 $this->createMock(EventDispatcherInterface::class),
                 $this->createMock(HookManager::class),
             ])
-            ->onlyMethods(['upgradeMigration'])
+            ->onlyMethods(['upgradeMigration', 'log'])
             ->getMock()
         ;
         $this->moduleManager->method('upgradeMigration')->willReturn(true);
+        $this->moduleManager->method('log')->willReturn(true);
```

Comment:
NOt sure the mock is adapted since the method doesn't return anything

Reply:
removed, thanks